### DISCUSSION TITLE
GPU accleration

### DISCUSSION THREAD

**Discussion by eishwezin20** [tensorflow/decision-forests#193] [Q&A] [answered]
Are there any GPU acceleration options for training models with TensorFlow Decision Forests?

**Comment by rstz** [comment, marked as answer]
TF-DF currently does not offer GPU-accelerated training. See [this issue](https://github.com/tensorflow/decision-forests/issues/38) for some more discussion on this. If anyone has a high-impact use case for GPU training, please contact the team for more discussions.

_Note: Our team deleted a blatantly incorrect answer to this question_
